js/data-parsing/kraken.js

```diff
@@ -1,13 +1,11 @@
 // Request library to make api calls with
 const request = require('request');
 
-// Currency pair to use
-var pair = "XXBTZUSD";
-//var pair = "XETHXXBT";
 
 // Local variables
 var highbids;
 var lowasks;
+var dataPair;
```

Comment:
`currencyPair`?
In general, I advise using the word "data" as little as possible in variable naming. Everything on a computer is data, it's a generic word.